src/components/EarnDashboard/components/ProviderDetails/OpportunityRow.tsx

```diff
@@ -216,6 +221,7 @@ export const OpportunityRow: React.FC<
             subText={assetCellSubText}
             justifyContent='flex-start'
             isExternal={opportunity.isReadOnly}
+            opportunityName={isRunePool ? opportunity.opportunityName : undefined}
```

Comment:
blocking: Not sure why this was ever optioinal but we should not need a ternary here - consistent use of `opportunityName` should be all we need as far as making rows consistent

<img width="499" alt="image" src="https://github.com/user-attachments/assets/87f79881-53bf-4698-aac3-8eb6110b8116">

Reply:
Even less diff by leveraging `showAssetSymbol` in [0ec463b](https://github.com/shapeshift/web/pull/7540/commits/0ec463b88d224c917700712c9b027ac53450b68b)